### DISCUSSION TITLE
Postcss & autoprefixer

### DISCUSSION THREAD

**Discussion by noahsb** [saadeghi/daisyui#1824] [Q&A] [answered]
Are `postcss` & `autoprefixer` packages required for daisyui? I see them listed as peer dependencies but not in the installation guide. Thanks!!

**Comment by saadeghi** [comment, marked as answer]
Post CSS is required. But if you have Tailwind CSS Installed, you already have Post CSS. It's a dependency for Tailwind CSS itself so no need to include it again in the docs.  
Auto prefixer is not required but it's recommend by Tailwind CSS. Ideally you would want your CSS to work on all browsers and Auto prefixer handles the vendor prefixes with no effort.  

Let me know if you have any questions
